Resources/Prototypes/_LostParadise/Recipes/Reactions/drinks.yml

```
@@ -0,0 +1,33 @@
+    Moonshine:
+      amount: 2
+    Mead:
+      amount: 1
+    Kvass:
+      amount: 1
+    JuiceLemon:
+      amount: 1
+    Gold:
+      amount: 1
+  products:
+    LPPCargoGargleBlaster: 6
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)